py/tests/external-deps/__main__.py

```diff
@@ -22,7 +22,3 @@
 from lib import greet
 print(f'\nFrom lib with wheel dependency: {greet("Matt")}')
 print(f'lib filepath: {inspect.getsourcefile(greet)}')
-
-from foo import get_branding
```

Comment:
This test is about external deps, will add other test coverage for internal deps.